player/tests/data/all.ron

```diff
@@ -2,7 +2,8 @@
 	backends: (bits: 0xF),
 	tests: [
 		"buffer-copy.ron",
-		"buffer-zero-init.ron",
```

Comment:
have you considered leaving this inside a single test? The test would have 2 expectations: one for the mapping, one of the binding. It can use different buffers for these.

Reply:
sure, can do! Personally, I prefer more atomic unit tests but hard to get anything like that anyways. I'll bundle everying buffer-zero-init related then